connexion/apis/aiohttp_api.py

```diff
@@ -167,7 +167,7 @@ def get_request(cls, req):
         logger.debug('Getting data and status code',
                      extra={'has_body': req.has_body, 'url': url})
 
-        query = {k: ','.join(v) for k, v in parse_qs(req.rel_url.query_string).items()}
+        query = {query_key: query_values for query_key, query_values in parse_qs(req.rel_url.query_string).items()}
```

Comment:
can this just be `query = parse_qs(req.rel_url.query_string)`?

Reply:
Yeah, nice catch ;-).